src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_TASK = 100;
+
     public static void main(String[] args) {
+        printWelcomeMessage();
+        boolean isExit = false;
+        Task[] list = new Task[MAX_TASK];
```

Comment:
The name "list" might be vague. Maybe consider renaming it? Perhaps something like "taskList" would be good